src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class OperationStrategyImplTest {
+    private static Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;
+    private static OperationStrategy operationStrategy;
+    private static OperationHandler balanceOperationHandler;
+    private static OperationHandler supplyOperationHandler;
+    private static OperationHandler purchaseOperationHandler;
+    private static OperationHandler returnOperationHandler;
+
+    @BeforeAll
+    static void beforeAll() {
+        operationHandlerMap = new HashMap<>();
+        operationStrategy = new OperationStrategyImpl(operationHandlerMap);
+        balanceOperationHandler = new BalanceOperationHandler();
+        supplyOperationHandler = new SupplyOperationHandler();
+        purchaseOperationHandler = new PurchaseOperationHandler();
+        returnOperationHandler = new ReturnOperationHandler();
+    }
+
+    @AfterEach
+    void tearDown() {
+        operationHandlerMap.clear();
+    }
+
+    @Test
+    void get_checkOnBalanceOperationHandler_Ok() {
+        operationHandlerMap.put(FruitTransaction.Operation.BALANCE, balanceOperationHandler);
+        OperationHandler actual = operationStrategy.get(FruitTransaction.Operation.BALANCE);
+        assertEquals(balanceOperationHandler,actual);

```

Comment:
```suggestion
        assertEquals(BalanceOperationHandler.class, actual);
```
no need to create an instance of each handlers